aospy/calc.py

```diff
@@ -397,21 +400,11 @@ def _create_input_data_obj(self, var, start_date=False,
         # specify what method to call to access the files on the filesystem.
         dmget(paths)
         ds_chunks = []
-        # 2015-10-16 S. Hill: Can we use the xarray.open_mfdataset function here
-        # instead of this logic of making individual datasets and then
-        # calling xarray.concat?  Or does the year<1678 logic make this not
-        # possible?
-
-        # 2015-10-16 19:06:00 S. Clark: The year<1678 logic is independent of
-        # using xarray.open_mfdataset. The main reason I held off on using it
-        # here was that it opens a can of worms with regard to performance;
-        # we'd need to add some logic to make sure the data were chunked in a
-        # reasonable way (and logic to change the chunking if need be).
         for file_ in paths:
             test = xr.open_dataset(file_, decode_cf=False,
```

Comment:
Unrelated, but during the refactor we definitely need to rename this variable within the code (this was my doing in the very beginning...)